plugins/azure-devops/README.md

```diff
@@ -63,13 +63,21 @@ spec:
 
 #### Mono repos
 
-If you have multiple entities within a single repo, you will need to specify which pipelines belong to each entity.
+If you have multiple entities within a single repo, you will need to specify which pipelines belong to each entity:
 
 ```yaml
 dev.azure.com/project-repo: <my-project>/<my-repo>
 dev.azure.com/build-definition: <build-definition-name>
 ```
 
+...and which README file belongs to each entity.
+
+Example:
```

Comment:
I would change these two lines to one to be something like this:

```
Then to display the `README` file that belongs to each entity you would do this:
```

Reply:
Done

Reply:
Can we remove line 73 and then change line 75 to be: "Then to display the `README` file that belongs to each entity you would do this:", please?

Reply:
Sorry, I thought I'd already done that. Must have missed pushing the commit.

Done now.